### ISSUE TITLE
Goto definition stopped working

### ISSUE THREAD

**Issue by Muzza** [slynch8/10x#1466]
See code below. Go to definition used to switch between these. Currently it is only going one way.
GotoFunctionImplementation is false

```cpp

namespace A
{
namespace B
{

class Thing
{
	int x;
	void func();	// <-- goto Definition does nothing here
}

};
};


namespace A
{
namespace B
{

void Thing::func( void )	// <-- goto Definition works here
{
}

};
};
```

**Comment by slynch8** [comment]
it's because it thinks the function has one arg (void) and doesn't match the declaration. I'll get that fixed

**Comment by slynch8** [comment]
fixed (0.36.3)
